app/services/terms_of_use_support.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+# Helper class for terms of use
+class TermsOfUseSupport
+  def self.full_statement(custom_rights_statement:)
+    [custom_rights_statement, I18n.t('license.terms_of_use')].compact_blank.join("\n\n")
+  end
+end
```

Comment:
This makes me wonder how we feel about helpers, generally. Helpers are rails-y, and we want to be "on the rails." But we've also intentionally avoided helpers in the past (maybe in the early H2 days?). But... this is an awfully tiny class, the documentation of which calls it out as a helper, and we do already use some helpers in H3. It may be worth chatting about when to use and when to avoid helpers in H3.

Reply:
Everything I know about helpers, I learned from jcoyne: Helpers are for use in views only.

Since `full_statement` isn't used exclusively in a view I didn't make it a helper.

As far as I know, Rails doesn't have a convention for a non-view helper. I've adopted my own convention of calling them "support", e.g., "TermsOfUseSupport".

Reply:
I bet we could come up with a way to write a module containing all the Rails helpers, which could then be included in various POROs (services, mappers, presenters), but I'm not sure that's better than what we have here. :shipit: